docs/content/en/docs/getting-started.md

```diff
@@ -29,6 +29,23 @@ Start integrating GoodData into your Python application right now.
     sdk = GoodDataSdk.create(host, token)
     ```
 
+    Alternatively, you can create the instance from a config file:
+
+    ```python
+    # The path to the profiles.yaml file
+    profiles= Path("~/.gooddata/profiles/Profile.yaml")
+    # Profile name defaults to `default`
+    sdk = GoodDataSdk.create_from_profile(profile="default", profiles_path=profiles)
```

Comment:
Change it, please, to this:

# The default path to the profiles.yaml file is set to `~/.gooddata/profiles/profiles.yaml`, but it can be overridden with a custom path
profiles= Path("~/my_directory/my_profiles.yaml")
# Profile name defaults to `default`, but it can be overridden with a custom profile
sdk = GoodDataSdk.create_from_profile(profile="staging", profiles_path=profiles)

Reply:
It is too long, I'll split the first part into two